### PR TITLE
Feat: Helm generator

### PR DESCRIPTION
**Pull request by MrPink**
To test:

`pip3 install -i https://test.pypi.org/simple/ aether-helm-generator`

`aether-helm-generator -d test.gather-data.org --template-path /tmp/ -p test --cloud-platform AWS --sg-id alb-ingress-controller.prod.k8s.eha.im -e prod --acm arn:aws:acm:eu-west-1:387526361725:certificate/8fbe428f --modules 'odk' --gather yes -db db.eu-west-1.rds.amazonaws.com -sb filesystem --secrets-path /tmp/
`

Suggestions for improvement very much welcome 🍕

### CODE REVIEW

File: helm-generator/helm_generator/secretsGenerator.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+import secrets
+
+
+def generate_secret():
+    """Generate secure token."""
+    secret = secrets.token_urlsafe(nbytes=36)
```
Comment (femilofin) [inline helm-generator/helm_generator/secretsGenerator.py]: secret not converted to base64

File: helm-generator/helm_generator/secretsGenerator.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    return(secret)
+
+
+def generate_secrets(application):
+    """Generate secrets."""
+    secrets_dict = {}
+    secrets_list = [
```
Comment (femilofin) [inline helm-generator/helm_generator/secretsGenerator.py]: missing `readonly-db-password`

File: helm-generator/helm_generator/args.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        ['-t', '--template-path', 'Specify template path', True],
+        ['-s', '--secrets-path', 'Specify template path', True],
+        ['-a', '--application', 'Specify the application', True],
+        ['-p', '--project', 'Specify a project name', True],
+        ['-c', '--cloud-platform', 'Specify the platform used', True],
+        ['-db', '--database-host', 'Specify the database host', True],
+        ['-sbn', '--storage-bucket-name', 'Specify storage bucket name', False],
+        ['-sb', '--storage-backend', 'Specify storage backend s3/gcp/filesystem', False],
+        ['--acm', '--aws-cert-arn', 'Specify AWS ACM', False],
+        ['--alb', '--alb-ingress-controller', 'True or False', False],
```
[the remark this answers is not in the document]
Comment (MrPink) [reply helm-generator/helm_generator/args.py]: yeah I was thinking about support for bare metal (which would be Nginx) but maybe I should change `cloud provider` to `platform` or similar and then derive the ingress from there?

File: helm-generator/helm_generator/args.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+import argparse
+
+
+def arg_list():
+    """Arg in a dict."""
+    arg_list = [
+        ['-d', '--domain', 'Specify the domain you are using', True],
+        ['-t', '--template-path', 'Specify template path', True],
+        ['-s', '--secrets-path', 'Specify template path', True],
+        ['-a', '--application', 'Specify the application', True],
```
[the remark this answers is not in the document]
Comment (MrPink) [reply helm-generator/helm_generator/args.py]: I was going to do a run per a service but passing a list of services is nicer, I'll update it 👍

File: helm-generator/helm_generator/args.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (adamvert) [reply helm-generator/helm_generator/args.py]: That sounds good - I guess you could leave out bare metal for now, but at least prepare the way for it in a future version of the script

**Pull request merged**